tests/run_tests

```diff
@@ -1,8 +1,18 @@
-#!/bin/sh
+#!/bin/bash
+set -e
 
 (cd tests; docker-compose up --build -d)
-printf "Sleeping for 10s until everything initializes...\n"
-sleep 10
-(cd tests; docker-compose exec api bin/db-upgrade)
 
-DISABLE_LOGS=1 bin/test
+printf "Waiting until db initializes..."
+until [ "` (docker inspect $(cd tests; docker-compose ps -q db) -f {{.State.Health.Status}})`" == "healthy" ]; do
```

Comment:
you could do a docker exec and use mysql to try and reach the db - if it's reachable it's safe(r) to assume that the DB is now available. Not going to block this PR for that though, your call.

Reply:
`{{.State.Health.Status}})` depends on `healthcheck` command in *tests/docker-compose.yml*, which is `mysqladmin ping ...`, so it's actually doing a similar thing :)